src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,50 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.domain.SessionImage.SessionImage;
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.NsUser;
+
+public class Session {
+    private Students students;
+    private Premium premium;
+    private SessionState sessionState;
+    private SessionImage sessionImage;
+
+    private int maxStudentCount;
+    private SessionDate sessionDate;
```

Comment:
```suggestion
    private SessionState state;
    private SessionImage image;

    private int maxStudentCount;
    private SessionDate date;
```

Session 객체 내 `session` prefix는 중복적인 의미로 해석할 수 있어보여요
객체 맥락에 따라 어느정도 생략이 가능하다고 생각합니다.

Reply:
재 생각에도 세션 객체 안에 존재하는 state, image 들이라 유추할 수 있을 것 같습니다.
수정 완료했습니다!